contracts/libs/SafeERC20WithPermit.sol

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+
+pragma solidity 0.7.6;
+
+import "../interfaces/IERC20WithPermit.sol";
+import "@openzeppelin/contracts/math/SafeMath.sol";
+import "@openzeppelin/contracts/utils/Address.sol";
+
+/**
+ * @title SafeERC20WithPermit
+ * @dev Wrappers around ERC20 operations that throw on failure (when the token
+ * contract returns false). Tokens that return no value (and instead revert or
+ * throw on failure) are also supported, non-reverting calls are assumed to be
+ * successful.
+ * To use this library you can add a `using SafeERC20WithPermit for IERC20WithPermit;` statement to your contract,
+ * which allows you to call the safe operations as `token.safeTransfer(...)`, etc.
+ */
+library SafeERC20WithPermit {
+    using SafeMath for uint256;
```

Comment:
Does SafeMath have any effect here? I think not and it should be removed.

Reply:
Removed.